packages/calcite-design-tokens/.eslintrc.js

```diff
@@ -0,0 +1,89 @@
+module.exports = {
```

Comment:
This used to be a `.cjs` file right, was the change to `.js` intentional? It looks like `tsconfig.cjs.json` is no longer required if that's the case. However you'll need to handle this file somehow in the tsconfigs because we get the following linting error as is:

![image](https://github.com/Esri/calcite-design-system/assets/10986395/0f755e83-436b-4782-8ce3-748e0bbd1e12)

Reply:
technically the `"files": []` key in package.json is only compatible with `type: "commonjs"`. So I was trying to make everything work with that but it's something we can resolve later. Switched by to `"type": "module"`